test/integration/users.test.js

```diff
@@ -585,6 +574,40 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users/?id", function () {
+    it("Should return given user by id", function (done) {
+      chai
+        .request(app)
+        .get(`/users/?id=${userId}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("User returned successfully!");
+          expect(res.body.user).to.be.a("object");
+          expect(res.body.user).to.not.have.property("phone");
+          expect(res.body.user).to.not.have.property("email");
```

Comment:
you only checked the fields which were not present . what about the fields which will be present

Reply:
is it necessary to check all the fields, if yes then what fields need to be present.

Reply:
Yes I think its a good idea to test for the fields present. it makes the tests strong. regarding `what fields need to be present` you need to check what fields you have added for the user you are adding in the test.

Reply:
added fields for user in test.